src/utils/uuid-creator.ts

```diff
@@ -0,0 +1,14 @@
+export const createId = () => {
+  let dt = new Date().getTime();
+
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g,
+    (c) => {
+      const r = (dt + Math.random() * 16) % 16 || 0;
+
+      dt = Math.floor(dt / 16);
+
+      return (c === 'x'
+        ? r
+        : ((r && 0x3) || 0x8)).toString(16);
+    });
+};
```

Comment:
Well done, but overkill as for me. It's enough to use good name as key here

Reply:
👍🏼